include/zephyr/net/buf.h

```diff
@@ -1007,26 +1012,36 @@ struct net_buf_pool {
 
 /** @cond INTERNAL_HIDDEN */
 #if defined(CONFIG_NET_BUF_POOL_USAGE)
-#define NET_BUF_POOL_INITIALIZER(_pool, _alloc, _bufs, _count, _ud_size, _destroy) \
+#define _NET_BUF_POOL_RUNTIME_DEFINE(_name, _count)				\
+	static struct net_buf_pool_rt_ctx _net_buf_pool_rt##_name = {		\
+		.free = Z_LIFO_INITIALIZER((_net_buf_pool_rt##_name).free),	\
+		.lock = { },							\
+		.uninit_count = _count,						\
+		.avail_count = _count,						\
+	};
+
+#define NET_BUF_POOL_INITIALIZER(_pool, _alloc, _rt, _bufs, _count, _ud_size, _destroy) \
```

Comment:
Similarly you are also here modifying a stable API by adding `_rt`.

Reply:
😭